site/docs/.vitepress/configs/algolia/crawler.js

```diff
@@ -18,7 +18,7 @@ const getPageRank = (url: Location) => {
 
 new Crawler({
   appId: "RBF5Q0D7QV",
-  apiKey: "...",
+  apiKey: "33782ffb584887e3b8cdf9e760ea8e60",
```

Comment:
Please revert

Reply:
Why?

Reply:
It's the key from your installation, not the one grammY uses. It shouldn't be part of this repo.

Reply:
It's the same API we use in https://github.com/grammyjs/website/blob/main/site/docs/.vitepress/configs/algolia/index.ts

Reply:
Oh, then it's the frontend key. Either way, it cannot be used for the crawler since it doesn't have write access to the index.

Reply:
Okay

Reply:
In my case I have the same API key for both frontend and crawler.

Reply:
But not the write access / admin API key

Reply:
Huh, I don't 🫠

Reply:
No idea why. They once changed a lot of things and now I no longer dare to touch anything as long as things are working.